desktop/core/src/desktop/urls.py

```diff
@@ -202,6 +204,13 @@
 ]
 
 dynamic_patterns += [
+    re_path('api/schema',
+        SpectacularAPIView.as_view(permission_classes=(AllowAny,)),
```

Comment:
Why `permission_classes=(AllowAny,)`?

Reply:
removed it, this was to allow non logged in users to access the url, by default this is available. hence not required